examples/create_etch_existing_cast.py

```diff
@@ -17,6 +17,12 @@ def main():
     packet = CreateEtchPacket(
         name="Etch packet with existing template",
         signature_email_subject="Please sign these forms",
+        # URL where Anvil will send POST requests when server events happen.
+        # Take a look at https://www.useanvil.com/docs/api/e-signatures#webhook-notifications
+        # for other details on how to configure webhooks on your account.
+        # You can also use sites like webhook.site, requestbin.com or ngrok to
+        # test webhooks.
+        # webhook_url="https://my.webhook.example.com/etch-events/
```

Comment:
Think this is _not_ supposed to be commented out? Or what's the pattern now? I see some more arguments that you've commented out as well

Reply:
Yeah I commented them out this pass since they're extra options with placeholders and users probably won't want these enabled by default. Tried to make this as usable as possible as-is once cast ids and other vars are replaced.